File: ai_generated_photos/v2/get/java/AIGeneratedPhotos.java
import com.amadeus.Amadeus;
import com.amadeus.Params;
import com.amadeus.exceptions.ResponseException;
import com.amadeus.resources.GeneratedPhoto;

public class AIGeneratedPhotos {

  public static void main(String[] args) throws ResponseException {

    Amadeus amadeus = Amadeus
        .builder("YOUR_AMADEUS_API_KEY","YOUR_AMADEUS_API_SECRET")
        .build();

    // Generates a fake mountain image
    GeneratedPhoto photo = amadeus.media.files.generatedPhotos.get(Params
        .with("category", "BEACH"));

    if (photo.getResponse().getStatusCode() != 200) {
        System.out.println("Wrong status code: " + photo.getResponse().getStatusCode());
        System.exit(-1);
    }

    System.out.println(photo);
  }
}

File: ai_generated_photos/v2/get/python/ai_generated_photos.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    Generates a photo with mountain
    '''
    response = amadeus.media.files.generated_photos.get(category='MOUNTAIN')
    try:
        import urllib.request
        urllib.request.urlretrieve(response.data['attachmentUri'], 'generated_image.png')
    except ImportError:
        import urllib
        urllib.urlretrieve(response.data['attachmentUri'], 'generated_image.jpg')
except ResponseError as error:
    raise error
